lib/Dialect/TTNN/Transforms/TTNNFusing.cpp

```diff
@@ -118,9 +115,12 @@ class TTNNFusingPass : public impl::TTNNFusingBase<TTNNFusingPass> {
 
   void runOnOperation() final {
     RewritePatternSet patterns(&getContext());
-    patterns.add<TTNNConv2dWithActivation<ReluOp>,
-                 TTNNConv2dWithActivation<Relu6Op>,
-                 TTNNConv2dWithActivation<SiluOp>>(&getContext());
+    // TODO: @mvasiljevicTT Add HardsigmoidOp once tt-metal issue is resolved
```

Comment:
:warning: **google\-readability\-todo** :warning:
missing username/bug in TODO
```suggestion
    // TODO(nsmith): @mvasiljevicTT Add HardsigmoidOp once tt-metal issue is resolved
```